jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/api/client/clientrequestcontext/JAXRSClientIT.java

```diff
@@ -860,6 +861,53 @@ protected void checkFilterContext(ClientRequestContext context)
     assertContains(entity, "Accept");
   }
 
+  /*
+   * @testName: containsHeaderStringTest
+   * 
+   * @assertion_ids:  JAXRS:JAVADOC:1353; JAXRS:JAVADOC:1354; 
+   * 
+   * @test_Strategy: Check if the specified header contains a specified value.
+   * 
+   * ClientRequestFilter.abortWith
+   */
+  @Test
+  public void containsHeaderStringTest() throws Fault {
+    ContextProvider provider = new ContextProvider() {
+      @Override
+      protected void checkFilterContext(ClientRequestContext context) throws Fault {
+          assertTrue(context.containsHeaderString("header1", "value"::equalsIgnoreCase));
```

Comment:
As "value" is already in the same case, this test for ignoring the case in not effective.

Reply:
Agreed.  Changed to ::equals